Makefile

```
@@ -2,14 +2,13 @@
 TMP_COMMIT_MSG_FILE=/tmp/commit_message.txt
 
 all: $(CURDIR)/src/node_modules
-	cd $(CURDIR)/src &&     \
-	gitbook build &&        \
-	rm -r $(CURDIR)/docs && \
-	mv $(CURDIR)/src/_book $(CURDIR)/docs
 
 $(CURDIR)/src/node_modules:
 	cd $(CURDIR)/src && \
-	gitbook install
+	npx honkit build --reload && \
+	cd .. && \
+	rm -r $(CURDIR)/docs && \
```

Optional: 既存部分ですが f オプションを付けておいたほうが万が一docsフォルダがなくてもエラーにならなくて嬉しいです。

```suggestion
	rm -rf $(CURDIR)/docs && \
```

---

ありがとうございます 取り込みました https://github.com/access-company/kotlin_intro/pull/44/commits/978af7743fde5a279f44b52dc24cc5e1c5ecc900